test/functional/flowcontrol/otel_format_test.go

```diff
@@ -0,0 +1,90 @@
+// go:build !fluentd
```

Comment:
This is a normalization test and should go there.

Reply:
This needs a bit of re-work so it will look different from this.